Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies/Helpers/CreateBericht.cs

```diff
@@ -101,6 +101,26 @@ public static async Task<Bericht> CreateBericht(this IPage page, CreateBerichtRe
                 BerichtType = request.BerichtType,
             };
         }
+        public static async Task OnSaveBericht(this IPage page, Bericht? bericht = null)
```

Comment:
UpdateBericht would be a name more inline with CreateBericht. 
The classname isn't adequate anymore either. should be something like ManageBericht